XLS-0070d-credentials/README.md

```diff
@@ -349,8 +332,8 @@ Note: the transaction will still fail if too many credentials are included. The
 
```

Comment:
>  `CredentialIDs` is included, but the destination doesn't have Deposit Authorization set up, or the transaction doesn't have a destination (e.g. `AccountSet`).

`CredentialIDs` is included, `Destination` have `lsfDepositAuth` set and  `Destination` not equal to  `Source` and   `DepositPreauth` object doesn't exist for the provided set of Credentials

Reply:
I don't understand what you mean by this - have my updates resolved this?